README.md

```diff
@@ -1 +1,121 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="porag-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Hello Microverse Project</b></h3>
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖  <a name="about-project">Hello_World</a>
+
+**Hello_World** is is the first project in the Microvers full stack full time program. The goal of this project is to demonstrate the standard flow of project completion basic with git, GitHub and coding best practices. 
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a> 
+<details>
+  <summary>Web Technologies</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **N/A**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+N/A
+
+### Setup
+N/A
+
+### Install
+N/A
+
+### Usage
+N/A
+
+### Run tests
+N/A
+
+### Deployment
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
- [ ] [REQUIRED] It would be descriptive if you remove `-s` end of `Authors` word 👍🏼